src/server/scripts/Outland/TempestKeep/botanica/boss_thorngrin_the_tender.cpp

```diff
@@ -64,6 +62,12 @@ class boss_thorngrin_the_tender : public CreatureScript
         void Reset() override
         {
             _Reset();
+            SetHealthCheckEvent(20, [&]() {
+                Talk(SAY_20_PERCENT_HP);
+            });
+            SetHealthCheckEvent(50, [&]() {
+                Talk(SAY_50_PERCENT_HP);
```

Comment:
why do we want it in Reset and not UpdateAI?

Reply:
It only needs to be setup/created once